inst/shiny/server.R

```diff
@@ -39,14 +39,14 @@ shinyServer(function(input, output, session) {
     
     updateCheckboxGroupInput(session, "area", 
                              choices = choices, 
-                             selected = choices)
+                             selected = choices[1])
```

Comment:
It's a bit putzy which is why I didn't implement immediately, but certainly could at some point.